shared/src/main/scala/org/jetbrains/sbt/structure/dataSerializers.scala

```diff
@@ -187,22 +187,44 @@ trait DataSerializers {
 
   implicit val projectDependencySerializer: XmlSerializer[ProjectDependencyData] = new XmlSerializer[ProjectDependencyData] {
```

Comment:
(minor) Naming:
Now we have two very similar names:
`projectDependencySerializer`
`projectDependenciesSerializer`

Let's rename the first one to `projectDependencyDataSerializer`